service/frontend/dcRedirectionPolicy.go

```diff
@@ -38,15 +35,18 @@ const (
 	DCRedirectionPolicyDefault = ""
 	// DCRedirectionPolicyNoop means no redirection
 	DCRedirectionPolicyNoop = "noop"
-	// DCRedirectionPolicySelectedAPIsForwarding means forwarding the following APIs based domain
+	// DCRedirectionPolicySelectedAPIsForwarding means forwarding the following non-worker APIs based domain
 	// 1. StartWorkflowExecution
 	// 2. SignalWithStartWorkflowExecution
 	// 3. SignalWorkflowExecution
 	// 4. RequestCancelWorkflowExecution
 	// 5. TerminateWorkflowExecution
 	// 6. QueryWorkflow
-	// please also reference selectedAPIsForwardingRedirectionPolicyWhitelistedAPIs
+	// 7. ResetWorkflow
+	// please also reference selectedAPIsForwardingRedirectionPolicyAPIAllowlist
 	DCRedirectionPolicySelectedAPIsForwarding = "selected-apis-forwarding"
+	// DCRedirectionPolicyAllDomainAPIsForwarding means forwarding all the worker and non-worker APIs based domain
+	DCRedirectionPolicyAllDomainAPIsForwarding = "all-domain-apis-forwarding"
```

Comment:
what about all-apis-forwarding? all-domains seems only domain related apis?

Reply:
yeah we don't(probably shouldn't) forward non-domain related api. 
For example, describe shard, add search attributes, etc